Extension/src/LanguageServer/ui.ts

```diff
@@ -168,6 +168,25 @@ export class UI {
             });
     }
 
+    public showCompileCommands(paths: string[]): Thenable<number> {
+        let options: vscode.QuickPickOptions = {};
+        options.placeHolder = "Select a compile_commands.json...";
+
+        let items: IndexableQuickPickItem[] = [];
+        for (let i: number = 0; i < paths.length; i++) {
+            items.push({label: paths[i], description: "", index: i});
+        }
+        items.push({ label: "Edit Configurations...", description: "", index: paths.length });
+
+        return vscode.window.showQuickPick(items, options)
+        .then(selection => {
```

Comment:
nit: please indent the `then` block to match coding style elsewhere in the file.